function-app/adb-to-purview/src/Function.Domain/Models/Parser/Settings/AppConfigurationSettings.cs

```diff
@@ -22,12 +25,12 @@ public class AppConfigurationSettings
         public string EventHubConsumerGroup { get; set; } = "read";
         public bool usePurviewTypes { get; set; } = false;
         public bool useResourceSet { get; set; } = true;
-        public string AuthEndPoint { get; set; } = "https://login.microsoftonline.com/";
+        public string AuthEndPoint { get; set; } = "https://login.microsoftonline.com/{0}";
         public string Authority
         {
             get
             {
-                return string.Format(CultureInfo.InvariantCulture, AuthEndPoint+"{0}", TenantId);
+                return string.Format(CultureInfo.InvariantCulture, AuthEndPoint, TenantId);
```

Comment:
Can we change this back on line 28 and 33?  It doesn't really make sense to include this as part of the configuration, right?